ins-app/app/src/main/java/ins/developers/indoornavigationsystem/MainActivity.java

```diff
@@ -0,0 +1,71 @@
+package ins.developers.indoornavigationsystem;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.JsonObjectRequest;
+import com.android.volley.toolbox.Volley;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class MainActivity extends AppCompatActivity {
+    final String TAG = this.getClass().getSimpleName();
+    TextView responseText;
+    Button getRequestButton;
+    EditText employeeSelector;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        responseText = (TextView) findViewById(R.id.result);
+        getRequestButton = (Button) findViewById(R.id.get_button);
+        employeeSelector = (EditText) findViewById(R.id.node_setter);
+
+        getRequestButton.setOnClickListener(new View.OnClickListener() {
+            public void onClick(View v) {
+                // Instantiate the RequestQueue.
+                RequestQueue queue = Volley.newRequestQueue(getApplicationContext());
+                String selectedEmployee = employeeSelector.getText().toString();
+                String employeeId = selectedEmployee.length() != 0 ? selectedEmployee : "1"; // Default employee id
```

Comment:
Maybe we need to pre-fetch all employee-id from server to the app. By that, we can have auto-complete or related feature which  ensure only valid values can be set.

Reply:
Sure!